### ISSUE TITLE
inverse cwt

### ISSUE THREAD

**Issue by renskegelderloos** [regeirk/pycwt#30]
I believe there is a square root missing (sj) in the calculation of the inverse wavelet transform. It should be:

iW = (dj * np.sqrt(dt) / wavelet.cdelta * wavelet.psi(0) *
          (np.real(W) / np.sqrt(sj)).sum(axis=0))

**Comment by jorgman1** [comment]
Yes, and a parenthesis around the first denominator. See pull request #29.
